util/test/checkChplInstall

```diff
@@ -104,6 +104,15 @@ log_info "Temporary test job directory: ${TMP_TEST_DIR}"
 function cleanup_tmp_dir()
 {
     log_debug "Removing ${TMP_TEST_DIR}"
+    local i
+    # sometimes on an NFS file system, rm needs to wait for the TEST_JOB exe file to close
+    for i in 1 2 3 4 5
+    do
+        rm -rf ${TMP_TEST_DIR} && break
+        log_warning "failed rm -rf ${TMP_TEST_DIR}, wait and try again"
+        ( set -x; ls -la ${TMP_TEST_DIR} ) || : ok
+        sleep 1
+    done
     rm -rf ${TMP_TEST_DIR} || (
```

Comment:
If rm -rf fails 5 times, the loop ends. 
In that case, that final rm -rf will produce an error message that says we give up now. Thats all its good for.